database.py

```diff
@@ -0,0 +1,79 @@
+import peewee
+
+class Result:
+    def __init__(self, is_success=True, data=None):
+        self.is_success = is_success
+        self.data = data
+
+class Database:
```

Comment:
Have / how has this been tested?

Reply:
It hasn't. I can test it and continue development in a week or two, or someone else can do it so it can be merged.

Reply:
I ran the (newly added) unit tests on a local db